zellij-server/src/wasm_vm.rs

```diff
@@ -248,6 +256,20 @@ fn host_set_timeout(plugin_env: &PluginEnv, secs: f64) {
     });
 }
 
+fn host_exec_cmd(plugin_env: &PluginEnv) {
```

Comment:
This function, here on the host-side, needs to check `plugin_env._allow_exec_host_cmd` before running anything!

Reply:
If permission is denied, I'd just use `log::warn()` to tell the user about it and return immediately. We probably don't need to crash and logging a warning + ignoring the request should do for now.